classes/mock/generator.php

```diff
@@ -201,6 +201,26 @@ public function generate($class, $mockNamespace = null, $mockClass = null)
 
 		return $this;
 	}
+	
+	public function getNewMockInstance($class, $mockNamespace = null, $mockClass = null, array $constructorArguments = array())
```

Comment:
So, I should create a method that will return "$mockNamespace . '\\' . $mockClass" to let the generator generate the class and create the instance in another place.